plugins/woocommerce/src/Blocks/BlockTypes/ProductStockIndicator.php

```diff
@@ -84,8 +84,13 @@ protected function render( $attributes, $content, $block ) {
 			return $content;
 		}
 
-		$post_id         = $block->context['postId'];
-		$product         = wc_get_product( $post_id );
+		$post_id = $block->context['postId'];
+		$product = wc_get_product( $post_id );
+
+		if ( ! $product ) {
+			return '';
```

Comment:
Should we return `null` ?

Reply:
The expected output is a string and in many other blocks we return an empty string in that case. So I think I'd stick to empty string.

Reply:
Works like a charm, thanks!

Reply:
Awesome! The fix will be available in WooCommerce 8.8 🙌

Reply:
Update: it will be included in WooCommerce 8.7! :)